db/seeds.rb

```diff
@@ -1,436 +1,175 @@
-    organisation: standalone_no_stock,
-    role: "data_provider",
-    is_dpo: true,
-  ) do |user|
-    user.password = ENV["REVIEW_APP_USER_PASSWORD"]
-    user.confirmed_at = Time.zone.now
-    create_data_protection_confirmation(user)
-  end
-
-  User.find_or_create_by!(
-    name: "Coordinator No Stock",
-    email: "coordinator.nostock@example.com",
-    organisation: standalone_no_stock,
-    role: "data_coordinator",
-  ) do |user|
-    user.password = ENV["REVIEW_APP_USER_PASSWORD"]
-    user.confirmed_at = Time.zone.now
-  end
-
-  User.find_or_create_by!(
-    name: "Stock owner 1",
-    email: "stock_owner1_dpo@example.com",
-    organisation: stock_owner1,
-    role: "data_coordinator",
-    is_dpo: true,
-  ) do |user|
-    user.password = ENV["REVIEW_APP_USER_PASSWORD"]
-    user.confirmed_at = Time.zone.now
-    create_data_protection_confirmation(user)
-  end
-
-  User.find_or_create_by!(
-    name: "Stock owner 2",
-    email: "stock_owner2_dpo@example.com",
-    organisation: stock_owner2,
-    role: "data_coordinator",
-    is_dpo: true,
-  ) do |user|
-    user.password = ENV["REVIEW_APP_USER_PASSWORD"]
-    user.confirmed_at = Time.zone.now
-    create_data_protection_confirmation(user)
-  end
-
-  User.find_or_create_by!(
-    name: "Managing agent 1",
-    email: "managing_agent1_dpo@example.com",
-    organisation: managing_agent1,
-    role: "data_coordinator",
-    is_dpo: true,
-  ) do |user|
-    user.password = ENV["REVIEW_APP_USER_PASSWORD"]
-    user.confirmed_at = Time.zone.now
-    create_data_protection_confirmation(user)
-  end
-
-  User.find_or_create_by!(
-    name: "Managing agent 2",
-    email: "managing_agent2_dpo@example.com",
-    organisation: managing_agent2,
-    role: "data_coordinator",
-    is_dpo: true,
-  ) do |user|
-    user.password = ENV["REVIEW_APP_USER_PASSWORD"]
-    user.confirmed_at = Time.zone.now
-    create_data_protection_confirmation(user)
-  end
-
-  OrganisationRelationship.find_or_create_by!(
-    parent_organisation: stock_owner1,
-    child_organisation: org,
-  )
-  OrganisationRelationship.find_or_create_by!(
-    parent_organisation: stock_owner2,
-    child_organisation: org,
-  )
-  OrganisationRelationship.find_or_create_by!(
-    parent_organisation: org,
-    child_organisation: managing_agent1,
-  )
-  OrganisationRelationship.find_or_create_by!(
-    parent_organisation: org,
-    child_organisation: managing_agent2,
+    housing_registration_no: nil,
+    rent_periods: all_rent_periods,
   )
 
   if Rails.env.development? || Rails.env.review?
-    User.find_or_create_by!(
-      name: "Provider",
-      email: "provider@example.com",
-      organisation: org,
-      role: "data_provider",
-    ) do |user|
-      user.password = ENV["REVIEW_APP_USER_PASSWORD"]
-      user.confirmed_at = Time.zone.now
-    end
+    stock_owner1 = FactoryBot.create(:organisation, :if_unique, :la, :holds_own_stock, name: "Stock Owner 1", rent_periods: all_rent_periods)
```

Comment:
I've kept names the same where people might be used to them, so we do have some fixed things to use.